docs/search/01-search-community-deploy/code_snippets/01_0306_configure_cert_manager_tls.sh

```diff
@@ -0,0 +1,159 @@
+required=(
+  K8S_CTX
+  MDB_NS
+  MDB_RESOURCE_NAME
+  MDB_MEMBERS
+  MDB_TLS_CA_SECRET_NAME
+  MDB_TLS_SERVER_CERT_SECRET_NAME
+  MDB_SEARCH_TLS_SECRET_NAME
+  CERT_MANAGER_NAMESPACE
+)
+missing=()
```

Comment:
if we're going with additional scripts or functions, then we could define them in env_variables.sh or another script that is sourced in env_variables.sh. 

The assumption is that sourcing env_variables.sh is a prerequisite step. But also let's remember that in the doc page we tell user "source the following env vars <include env_variables.sh>" 
we don't sa anything about "source also this script containing helper functions".